src/IBM.WatsonDeveloperCloud.SpeechToText.v1/SpeechToTextService.cs

```diff
@@ -2740,30 +2778,40 @@ public BaseModel TrainAcousticModel(string customizationId, string customLanguag
         /// custom model.</param>
         /// <param name="customLanguageModelId">If the custom acoustic model was trained with a custom language model,
         /// the customization ID (GUID) of that custom language model. The custom language model must be upgraded before
-        /// the custom acoustic model can be upgraded. (optional)</param>
+        /// the custom acoustic model can be upgraded. The credentials specified with the request must own both custom
+        /// models. (optional)</param>
+        /// <param name="force">If `true`, forces the upgrade of a custom acoustic model for which no input data has
+        /// been modified since it was last trained. Use this parameter only to force the upgrade of a custom acoustic
+        /// model that is trained with a custom language model, and only if you receive a 400 response code and the
+        /// message `No input data modified since last training`. See [Upgrading a custom acoustic
+        /// model](https://cloud.ibm.com/docs/services/speech-to-text/custom-upgrade.html#upgradeAcoustic). (optional,
+        /// default to false)</param>
         /// <param name="customData">Custom data object to pass data including custom request headers.</param>
         /// <returns><see cref="BaseModel" />BaseModel</returns>
-        public BaseModel UpgradeAcousticModel(string customizationId, string customLanguageModelId = null, Dictionary<string, object> customData = null)
+        public BaseModel UpgradeAcousticModel(string customizationId, string customLanguageModelId = null, bool? force = null, Dictionary<string, object> customData = null)
```

Comment:
Probably a silly question considering how many times I've reviewed this SDK, but adding the parameter here doesn't break anything, right? I know you can use named parameters in C# but wasn't sure if someone could pass in `customData` as the last param without specifying the name.

Reply:
Yeah this is technically a breaking change if they don't use named params. Let me move this after `customData`